docs/USER_GUIDE.md

```diff
@@ -0,0 +1,173 @@
+# Flint IDE User Guide  
+This is the MPS Flint IDE user guide. The Guide covers the following topics.
+- [Creating a Flint project](#creating-a-flint-project)
+- [Import existing flint model](#optional-import-existing-flint-model)
+- [Creating a Flint project](#creating-a-flint-project)
+- [Editing a FlintModel](#editing-a-flintModel)
+- [Creating a FlintRunner](#creating-a-flintRunner)
```

Comment:
The `Creating a Flint project` is listed twice

Reply:
On is creating flint project and one is creating flintrunner

Reply:
It is listed on both line 3 and 5